### DISCUSSION TITLE
Question: where is located the Dockerfile of the nerdctl-ipfs-registry image?

### DISCUSSION THREAD

**Discussion by aledbf** [containerd/nerdctl#962] [Q&A] [answered]
### What is the problem you're trying to solve

Update the image `ghcr.io/stargz-containers/nerdctl-ipfs-registry:v0.16.0`

### Describe the solution you'd like

Link to the Dockerfile to be able to update the image (maybe I just need nerdctl?)

**Comment by aledbf** [comment]
One missing thing in the image is `unpigz`

**Comment by ktock** [comment, marked as answer]
@aledbf 

> Link to the Dockerfile to be able to update the image (maybe I just need nerdctl?)

Yes, it just contains nerdctl.
 Dockerfile used to build that image is:

https://github.com/containerd/nerdctl/tree/v0.18.0/examples/nerdctl-ipfs-registry-kubernetes#example-dockerfile-of-nerdctl

```Dockerfile
FROM ubuntu:20.04
ARG NERDCTL_VERSION=0.16.0
RUN apt-get update -y && apt-get install -y curl && \
    curl -sSL --output /tmp/nerdctl.tgz https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-${NERDCTL_VERSION}-linux-${TARGETARCH:-amd64}.tar.gz && \
    tar zxvf /tmp/nerdctl.tgz -C /usr/local/bin/ && \
    rm /tmp/nerdctl.tgz
ENTRYPOINT [ "/usr/local/bin/nerdctl", "ipfs", "registry", "serve" ]
```

> One missing thing in the image is unpigz

Thank you for the suggestion.
We'll add it to the image and push it with the latest nerdctl.

**Reply by aledbf** [reply]
@ktock thank you for the quick response! :pray:
